src/main/scala/io/iohk/ethereum/network/p2p/messages/BaseETH6XMessages.scala

```diff
@@ -58,7 +58,7 @@ object BaseETH6XMessages {
   implicit val accessListItemCodec: RLPCodec[AccessListItem] =
     RLPCodec.instance[AccessListItem](
       { case AccessListItem(address, storageKeys) =>
-        RLPList(address, toRlpList(storageKeys))
+        RLPList(address, toRlpList(storageKeys.map(UInt256(_).bytes.toArray)))
```

Comment:
We should probably add a test to 
 - make the round trip and check that the serialization is the same with the padded zeros. 
 - check that we get the right sender from a given transaction (because it was wrong when we forgot to add the padded zeros)